pnpm-lock.yaml

```diff
@@ -14839,7 +14842,7 @@ packages:
     peerDependencies:
       webpack: ^5.0.0
     dependencies:
-      webpack: 5.75.0(webpack-cli@5.0.1)
+      webpack: 5.75.0(webpack-cli@4.10.0)
```

Comment:
I think we should revert this change?

Reply:
This change is not caused by my modification. I try to get the historical pnpm-lock.yml to overwrite and then install the dependencies, and this modification will be automatically brought. In addition, I see that there is already a dependency of `webpack 5(webpack-cli@4) in pnpm-lock.